### ISSUE TITLE
Check if BackUp Folder exists.

### ISSUE THREAD

**Issue by ghost** [bit-team/backintime#891]
There should be an option whether to check if the BackUp Folder actually exists or not. I had BackInTime set to write BackUps every hour, but the external drive wasn't connected once. BackInTime then just created the BackUpFolder /media/name/BackUp and whenever I mounted the drive it was mounted in /media/name/BackUp1 without me noticing. My autoremove settings then continued to always remove every but the last snapshot in the BackUp Folder because my drive is smaller than the external one. That lead to me basically not having BackUp's for the last month because I just now noticed. 
TL;DR: 
An option whether to create the BackUp Folder when it doesn't exist or not would be great!

**Comment by buhtzz** [comment]
No responses from "ghost" accounts.

I read this topic before but (sorry again) can't find that issue. I suggest to make a Feature Request from it to give the user a hint about when the destination is a mount point and not a folder under the mount point. Of course we have to think more about it what we can do. But this is an often upcoming "problem".

Tag: Feature, Low, Discussion
Maybe Close if we can find the other Issues related to that topic.

**Comment by emtiu** [comment]
Closing as duplicate of #1016.
